slyd/slyd/jsonapi/serializers.py

```diff
@@ -100,7 +101,7 @@ def __new__(mcs, name, bases, attrs):
         cls = super(JsonApiSerializerMeta, mcs).__new__(mcs, name, bases, attrs)
 
         # add new schema to registry by type
-        serializers[schema_type] = cls
+        schemas[schema_type] = cls
```

Comment:
Should be serializers (changing the comment as well). Marshmallow has the notion of schemas, but we are really using them as serializers (as portrayed by the meta class name `JsonApiSerializerMeta`.